src/Entity/EmbedButton.php

```diff
@@ -128,7 +128,7 @@ public function getIconFile() {
    */
   public function getIconUrl() {
     if ($image = $this->getIconFile()) {
-      return $image->url();
```

Comment:
I didn't try, but I think the toUrl() cast to a string will still return the file/fid path.